src/Libraries/GeometryUIWpf/PanelSurfaceBoundaryConditions.cs

```diff
@@ -0,0 +1,54 @@
+using Autodesk.DesignScript.Geometry;
+using Autodesk.DesignScript.Runtime;
+using CoreNodeModels;
+using DSCore;
+using Dynamo.Graph.Nodes;
+using ProtoCore.AST.AssociativeAST;
+using System;
+using System.Collections.Generic;
+
+namespace GeometryUIWpf
+{
+    [IsVisibleInDynamoLibrary(true)]
+    [NodeName("PanelSurfaceBoundaryConditions")]
```

Comment:
Can we just make this singular like the enum itself - `PanelSurfaceBoundaryCondition`?

Reply:
PanelSurfaceBoundaryCondition will collide with the ENum coming from ProtoGeom (at least in the library view)

Reply:
![image](https://github.com/DynamoDS/Dynamo/assets/46732933/5fa01b4a-fa02-4050-a447-27a2e204e5ca)

Reply:
@aparajit-pratap is the layout spec good enough for now ?

Reply:
If they have the same name (under the same parent) one will not be shown

Reply:
maybe we can fix that later. This is still experimental right ?

Reply:
or maybe there's a special setting to allow same name nodes under the same parent ?

Reply:
Or I can change the name of the new node (again)

Reply:
Maybe only the same named nodes are allowed if and only if they have different inputs?

Reply:
@Amoursol what do you think about the layout above ? 2 entries with the same name "PanelSurfaceBoundaryCOndition" 
ONe is a dropdown node, the other is category (I think) for individual boundaryCOndition nodes

Reply:
@pinzart90 As it's still experimental, I don't think layoutspec changes matter too much. So we can do any of those changes and evaluate later if need be. 

On the name front, could we please name it "Select Boundary Condition"? This is similar to some Revit paradigms. Do we also need to mark it an Action, not a Query? 

![image](https://github.com/DynamoDS/Dynamo/assets/12857357/855d762b-c1d0-4268-8060-689152647a31)